torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -80,12 +84,297 @@ struct TensorInfo {
   IndexType strides[N];
 };
 )");
+constexpr auto rand_support_literal = R"(
```

Comment:
It would be very good to mention in a comment above this WHY curand can't be used, so it's obvious to anyone reading this code. (IIUC, it's because nvrtc doesn't support curand, is that right? So if one day, nvrtc does support curand, we know we can get rid of this header.)